lib/memery.rb

```diff
@@ -25,6 +26,8 @@ def monotonic_clock
     end
   end
 
+  extend ModuleMethods
```

Comment:
This adds `method_visibility` and `monotonic_clock` to each module that includes Memery. I think we shold avoid that (and maybe even spec that).

Reply:
OK, returned to `Memery` itself. Thank you.